webpack_loader/loader.py

```diff
@@ -139,3 +139,19 @@ def get_bundle(self, bundle_name):
             "The stats file does not contain valid data. Make sure "
             "webpack-bundle-tracker plugin is enabled and try to run "
             "webpack again.")
+
+
+class FakeWebpackLoader(WebpackLoader):
```

Comment:
Since we have 2 classes in here, should I rename this module to `loaders` (plural)?

Reply:
plural is a better choice, although I'd also be fine with `webpackloader_exts` (okay, maybe that's too explicit)

Reply:
It's possible that projects import this module directly to inherit from `WebpackLoader` and build custom loaders. So to avoid breaking the API just for using a plural, I prefer to not change the module name.

Reply:
We can leave a module in singular form that imports everything from the plural and logs a deprecation warning

Reply:
>  although I'd also be fine with webpackloader_exts (<APP-NAME>_exts)(okay, maybe that's too explicit)

Result of reviewing pull-request right after waking up:
I was pretty sure this was being done in cookiecutter-django and not django-webpack-loader. My mistake, sorry. 
The `<APP-NAME>_exts` format makes sense to me for projects that extend `django-webpack-loader`, but doesn't make sense for `django-webpack-loader` itself